README.md

```diff
@@ -15,7 +15,7 @@ For the now-non-functional v1.x library, please [see the Legacy API branch](http
 Setting up Instafeed is pretty straight-forward - there are 3 main steps.
 
  1. Create a Facebook app linked to Instagram, and add yourself as a test user. See [Managing User Tokens](https://github.com/stevenschobert/instafeed.js/wiki/Managing-Access-Tokens).
- 2. Create an access token and provide it to an [Instagram Token service](https://github.com/companionstudio/instagram-token-agent)
+ 2. Create an access token and provide it to an [Instagram Token service](https://www.instant-tokens.com/home)
```

Comment:
Would you mind modifying this to say something like "or use instant-tokens.com"?

The phrasing in the current form doesn't make much sense, since instant-tokens actually provides the access token.

Thanks!